src/api/config/config.ts

```diff
@@ -0,0 +1,8 @@
+const config = {
+  wamp: {
+    realm: 'prices',
```

Comment:
could you plz expose configuration properties as env variables following this pattern https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-custom-environment-variables